webapp/src/domain/tests/Subscriptions.tsx

```diff
@@ -139,6 +147,19 @@ export default function Subscriptions(props: SubscriptionsProps) {
                     props.onModified(true)
                 }}
             />
+            <br />
+            <Divider />
+            <br />
+            <Checkbox
+                onChange={(checked, ev) => {
+                    setIsCheckedMissingRuns(checked)
+                    ev.currentTarget.value = checked.toString()
```

Comment:
Are you sure this line is necessary?

Btw., there's https://www.patternfly.org/v4/components/switch component (the choice is up to you, but I find Switch nicer when it's one option alone). Check out https://www.patternfly.org/v4/guidelines/usage-and-behavior#selecting-between-options-on-a-form

Reply:
ok I'll try and get this switched over.

Reply:
yes I found the code for onChange handler necessary. Removing it causes nothing to happen when the UI object is clicked.

Reply:
Fair enough, I can try to find out what's happening when this gets integrated.